packages/alert/src/props.tsx

```diff
@@ -7,6 +7,10 @@ export type AlertProps = {
    * Type of alert
    */
   type: 'negative' | 'positive' | 'warning' | 'info';
+  /**
+   * ARIA live region "role" attribute value
+   */
+  role: string;
```

Comment:
This should be optional.

```suggestion
  role?: string;
```